src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -69,12 +69,13 @@ export const KarmaScore = ({score}: KarmaScoreProps) => {
                 }}>
                 The higher your Karma score, the higher your visibility on the platform. Get
                 rewarded for positive contributions to the community - and avoid being blocked by
-                others!
+                others! Your Karma score will be shown as a circle around your profile picture,
+                including for your anonymous posts.
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of information about where the Karma score will be displayed is useful for users. It helps them understand where they can find their Karma score on the platform.
```diff
- others!
+ others! Your Karma score will be shown as a circle around your profile picture,
+ including for your anonymous posts.
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->